### PR TITLE
DEVDOCS-3294-add-409-error-status

### PR DESCRIPTION
**Pull request by traci707**
[DEVDOCS-3294](https://jira.bigcommerce.com/browse/DEVDOCS-3294)
added 409 status

### CODE REVIEW

**Comment by slsriehl** [comment]
409 description appears to be as requested.  The 400 description above has a leading space in the string literal.  Will suggest.

Looking at the [current live doc](https://developer.bigcommerce.com/api-reference/storefront/customers/customers/createacustomer) and the [source ticket CUST-1716](https://jira.bigcommerce.com/browse/CUST-1716) -- I wonder whether @filakhtov or @huntario wants to opine on including more of the expected behavior in the `409` tab of the `Responses` box.  

(also @traci707 @markcmurphy IDK how we'd write that in YAML to display in the error description display box, but surely there's a way).

File: reference/customers.sf.yml [hunk 1]
```diff
@@ -30,6 +30,8 @@ paths:
           description: Customer succesfully created
         '400':
           description: ' Could not create customer'
```
Comment (slsriehl) [inline reference/customers.sf.yml:32]:
```suggestion
          description: Could not create customer
```

**Comment by huntario** [comment]
> 


> including more of the expected behavior in the `409` tab of the `Responses` box
The current description seems good to me. For security reasons, I don't think we would want to go into the details outline in the comments of CUST-1716 about when SPAM protection is off, i.e. we should not reveal that they will stop getting a response if SPAM protection is off

**Review by slsriehl — APPROVED** [review]
LGTM

**Comment by filakhtov** [comment]
👋🏻  Hey @slsriehl, I am not sure what your question is here, can you please clarify?

**Comment by slsriehl** [comment]
> 👋🏻 Hey @slsriehl, I am not sure what your question is here, can you please clarify?

@filakhtov -- @huntario answered my question, above.

**Pull request merged**